src/Microsoft.Health.Dicom.Core/Models/RequirementCode.cs

```diff
@@ -35,4 +35,38 @@ public enum RequirementCode
     /// Optional for the SCU, and cannot be zero length.
     /// </summary>
     ThreeThree,
+    /// <summary>
+    /// Custom requirement code which is not part of the standard.
```

Comment:
Let's update the comment here and below to indicated that these are special requirements that do not have a code in the table - they are still part of the standard.